dispatcher_plugin_nb2workflow/queries.py

```diff
@@ -160,26 +177,34 @@ def process_product_method(self, instrument, prod_list, api=False):
             query_out.prod_dictionary['binary_data_product_list'] = bin_dp_list
             query_out.prod_dictionary['binary_image_product_list'] = bin_im_dp_list
             query_out.prod_dictionary['text_product_list'] = text_dp_list
+            query_out.prod_dictionary['progress_product_list'] = progress_dp_list
         else:
-            prod_name_list, file_name_list, image_list = [], [], []
+            prod_name_list, file_name_list, image_list, progress_product_list = [], [], [], []
             for product in prod_list.prod_list:
-                product.write()
-                try:
-                    file_name_list.append(os.path.basename(product.file_path))
-                except AttributeError:
-                    pass
-                im = product.get_html_draw()
-                if im:
-                    image_list.append(im)
+                html_draw = product.progress_data
```

Comment:
Something wrong here, this should be for ProgressProduct only

And now there is no call for `get_html_draw()` of other products

Reply:
yes, I am fixing it now

Reply:
Hm, but tests have passed. Proper html image generation for normal products is not covered. Added an issue https://github.com/oda-hub/dispatcher-plugin-nb2workflow/issues/77